src/server/api/routers/config.ts

```diff
@@ -223,4 +226,21 @@ export const configRouter = createTRPCRouter({
       const targetPath = path.join('data/configs', `${input.name}.json`);
       fs.writeFileSync(targetPath, JSON.stringify(newConfig, null, 2), 'utf8');
     }),
+  // publicProcedure is not optimal, but should be fince, since there is no input and output data nor can you break the config
+  updateConfigurationSchemaToLatest: publicProcedure.mutation(async () => {
```

Comment:
Is that then only executed on new installations? What do we do with existing ones?

Reply:
It is only executed, if the version was created with a version, that didn't have the DB mount.
It's a janky solution, but works well with existing instances and new instances.
We don't do anything with the schema version so far, so overwriting should be no problem.